lib/maths/CTimeSeriesSegmentation.cc

```diff
@@ -665,21 +673,20 @@ void CTimeSeriesSegmentation::fitPiecewiseLinearScaledSeasonal(
 
     auto computeScales = [&] {
         for (std::size_t j = 1; j < segmentation.size(); ++j) {
-            TMeanAccumulator projection;
-            TMeanAccumulator Z;
+            TMeanAccumulator xp;
+            TMeanAccumulator pp;
```

Comment:
Do the new variable names `xp` and `pp` refer to some formula? For me personally, it was easier to understand what `projection` and `Z` meant.

Reply:
It was just to make these consistent with other code which does a similar thing, i.e. I went for consistency. I'm happy to instead standardise on these names.

Reply:
Reverted in [079a076](https://github.com/elastic/ml-cpp/pull/1550/commits/079a07614ed72f3b9562f478b634eaedbcd68f77).